reading_comprehension/modules/depth_sep_conv.py

```diff
@@ -0,0 +1,59 @@
+import collections
```

Comment:
Even in the filename, I'd avoid abbreviations.  `conv` is probably fine; I'd recommend `depthwise_separable_conv.py`.  It's ok to have long names - better that than making the reader guess what you meant.